rpc/gnmi/protobuf-vals.md

```diff
@@ -0,0 +1,214 @@
+# Carrying Binary Wire Format Protobuf Messages in gNMI
+**Contributors**: robjs, csl  
+May 2018  
+*Implementation Status*: Proposal
+
+## Problem Statement
+For some applications the data carried by gNMI is directly described by a
```

Comment:
nit: add comma , "For some applications, the data ..."

Reply:
Done.